views/index.erb

```diff
@@ -96,8 +96,8 @@
                       ['sp', 'ServiceProvider configured'],
                       ['', 'Default'],
                       ['2', 'AAL2'],
-                      ['3', 'Phishing-resistant AAL2'],
-                      ['3-hspd12', 'HSPD12 required'],
+                      ['2-phishing_resistant', 'Phishing-resistant AAL2'],
+                      ['2-hspd12', 'HSPD12 required'],
```

Comment:
Does this affect the [constants in IDP](https://github.com/18F/identity-idp/blob/main/lib/saml_idp_constants.rb#L52)? I don't think it does, but want to confirm with someone else before saying its good.

Reply:
great question! No, those are used internally for the request that generates the sign-in request, but thanks for checking!